Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ccl/dccwalletinfo/storage/database/DccWalletInfoDatabaseTest.kt

```diff
@@ -0,0 +1,194 @@
+        titleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Impfstatus"),
+            parameters = listOf()
+        ),
+        subtitleText = PluralText(
+            type = "plural",
+            quantity = 25,
+            localizedText = mapOf(
+                "de" to QuantityText(
+                    zero = "Letzte Impfung heute",
+                    one = "Letzte Impfung vor %u Tag",
+                    two = "Letzte Impfung vor %u Tagen",
+                    few = "Letzte Impfung vor %u Tagen",
+                    many = "Letzte Impfung vor %u Tagen",
+                    other = "Letzte Impfung vor %u Tagen"
+                )
+            ),
+            parameters = listOf(
+                Parameters(
+                    type = Parameters.Type.DATE,
+                    value = "2021-12-01",
+                    format = Parameters.FormatType.DATE_DIFF_NOW,
+                    unit = Parameters.UnitType.DAY
+                )
+            )
+        ),
+        longText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Sie haben nun alle derzeit [...]"),
+            parameters = listOf()
+        ),
+        faqAnchor = "dcc_admission_state"
+    )
+
+    private val admissionState = AdmissionState(
+        visible = true,
+        badgeText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "2G+"),
+            parameters = listOf()
+        ),
+        titleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Status-Nachweis"),
+            parameters = listOf()
+        ),
+        subtitleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "2G+ PCR-Test"),
+            parameters = listOf()
+        ),
+        longText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Ihre Zertifikate erfüllen [...]"),
+            parameters = listOf()
+        ),
+        faqAnchor = "dcc_admission_state"
+    )
+
+    private val dccWalletInfo = DccWalletInfo(
+        admissionState = admissionState,
+        vaccinationState = vaccinationState,
+        verification = verification,
+        boosterNotification = boosterNotification,
+        mostRelevantCertificate = mostRelevantCertificate,
+        validUntil = Instant.parse("2022-01-14T18:43:00Z")
+    )
+
+    private val personIdentifier = CertificatePersonIdentifier(
+        firstNameStandardized = "Erika",
+        lastNameStandardized = "MusterFrau",
+        dateOfBirthFormatted = "1980-01-01"
+    )
+
+    @Before
+    fun createDb() {
+        val context = ApplicationProvider.getApplicationContext<Context>()
+        db = Room.inMemoryDatabaseBuilder(
+            context, DccWalletInfoDatabase::class.java
+        ).build()
+        dao = db.dccWalletInfoDao()
+    }
+
+    @After
+    @Throws(IOException::class)
```

Comment:
Why do we need `@Throws(IOException::class)` here?

Reply:
No necessary, but I would keep it , just in case the test fails because of it  , see https://developer.android.com/training/data-storage/room/testing-db